designer/submodules/packages/components/src/scenarios/list/item.tsx

```diff
@@ -52,9 +57,17 @@ export function FirstLine({ row }: { row: RowType }): JSX.Element {
     const filtersContext = useFilterContext<ScenariosFiltersModel>();
 
     return (
-        <CopyTooltip text={row.name} title={t("scenario.copyName", "Copy name to clipboard")}>
-            <Highlight value={row.name} filterText={filtersContext.getFilter("NAME")} />
-        </CopyTooltip>
+        <div style={{ display: "flex" }}>
+            <Category category={row.processCategory} filtersContext={filtersContext} />
+            <span style={{ paddingLeft: 8, paddingRight: 8 }}>/</span>
+            <CopyTooltip text={row.name} title={t("scenario.copyName", "Copy name to clipboard")}>
+                <Highlight
+                    value={row.name}
+                    filterText={filtersContext.getFilter("NAME")}
+                    style={{ fontWeight: "bold", fontSize: "1rem" }}
```

Comment:
I would say, instead of extending the interface only to handle one case, let's use the styled component and create the next Highlight variant

```
const HighlightedName = styled(Highlight)({
    fontWeight: "bold",
    fontSize: "1rem",
});
```

Reply:
Fixed.